lib/ssl/src/ssl_cipher.erl

```diff
@@ -1053,7 +1050,7 @@ filter_suites_pubkey(ecdsa, Ciphers, _, OtpCert) ->
                                    ec_ecdhe_suites(Ciphers)),
     filter_keyuse_suites(keyAgreement, Uses, CiphersSuites, ec_ecdh_suites(Ciphers)).
 
-filter_suites_signature(_, Ciphers, {3, N}) when N >= 3 ->
+filter_suites_signature(_, Ciphers, ?'TLS-1.X'=Version) when Version >= ?'TLS-1.2' ->
```

Comment:
@IngelaAndin I am guessing I can remove this first clause as well, as pointed out by your comments from below, correct?

Reply:
No here you should match on TLS-1.2. That is how I know that the function rsa_signed_suites will only be called for lower than TLS-1.2. Also I know that this code will never be called when the version is TLS-1.3.